docs/source/conf.py

```diff
@@ -155,3 +156,14 @@ def setup(sphinx):
 
 # -- Options for EPUB output
 epub_show_urls = "footnote"
+
+# -- Automatic fetch citation info from zenodo
+def generate_bibtex():
+    result = subprocess.run(["python", "_scripts/generate-bibtex.py"], capture_output=True, text=True)
+    return result.stdout.strip()
```

Comment:
Why does this need to run in a subprocess?

Reply:
Good question. Changed it, such that it is now used as a function